server/socket.js

```diff
@@ -0,0 +1,12 @@
+import { Server } from "socket.io";
+import logger from "./utils/logger";
+
+export let io = null;
+export const init = (server) => {
+	io = new Server(server, { path: "/api/socket.io" });
+	io.on("connection", (socket) => {
+		socket.on("test", function ({ requestId }) {
+			logger.info("received" + requestId);
```

Comment:
(-) In the future (when "test" has been renamed to "laptop_request:subscribe", this is where you add the socket to a room named for the requestId)

Reply:
cool